components/director/internal/domain/application/resolver_test.go

```diff
@@ -1016,7 +1018,9 @@ func TestResolver_SetApplicationLabel(t *testing.T) {
 		// then
 		require.Nil(t, result)
 		require.Error(t, err)
-		assert.EqualError(t, err, "validation error for type LabelInput: key: cannot be blank; value: cannot be blank.")
+		assert.Contains(t, err.Error(), "key=cannot be blank")
```

Comment:
I think error messages should be deterministic. Having asserts to check error message clutters the code, also, when user sends the same request to API while the state of application remains unchanged, he should always receive the same error, in my opinion.